test_scripts/RC/CLIMATE_RADIO/OnRemoteControlSettings/024_Allowed_false_no_PTU.lua

```diff
@@ -13,8 +13,7 @@
 -- SDL must:
```

Comment:
@LitvinenkoIra Please add link to updated req-t (as in line 4 in script 008)

Reply:
@dboltovskyi Done db789fa84af61fc45231bb4dbae583598fa4f55c